keras_nlp/models/albert/albert_group_layer.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from tensorflow import keras
+
+from keras_nlp.layers.transformer_encoder import TransformerEncoder
+
+
+class AlbertGroupLayer(keras.layers.Layer):
```

Comment:
It seems to me like we could remove this (and definitely preference to do so if we can). But I may be missing something here.